src/main/kotlin/solve/importer/view/ButtonView.kt

```diff
@@ -0,0 +1,85 @@
+package solve.importer.view
+
+import javafx.geometry.Insets
+import javafx.scene.control.Button
+import solve.importer.ProjectParser.createAlert
+import solve.main.MainController
+import solve.menubar.view.MenuBarView
+import solve.project.model.ProjectFrame
+import tornadofx.*
+
+
+class ButtonView : View() {
+
+    private val mainController: MainController by inject()
+
+    private val directoryPathView: DirectoryPathView by inject()
+
+    private fun cancelAction() {
+        MenuBarView().importer.close()
+    }
+
+    private fun importAction(button: Button) {
+        val projectVal = directoryPathView.project.value
+        try {
+            mainController.sceneFacade.visualize(projectVal.layers, projectVal.frames.map { it.frame })
+            mainController.displayCatalogueFrames(projectVal.frames.map { it.frame })
+            directoryPathView.project.set(null)
+            button.isDisable = true
+            MenuBarView().importer.close()
+        } catch (e: Exception) {
+            createAlert("Visualization error")
+        }
+    }
+
+    private val cancelButton = button("Cancel") {
+        action {
+            cancelAction()
+        }
+        prefWidth = 180.0
+    }
+
+    private val importButton = button("Import") {
+        isDisable = true
```

Comment:
Let's implement isDisable and other logic for buttons in MVC-style. Create model for them with required properties, some controller should update them depending on business-logic (file parsing), view just handles user input and displays UI